apis/ethereum/eth_sendRawTransaction.md

```diff
@@ -0,0 +1,40 @@
+# eth_sendRawTransaction
+
+
+> Creates a new message call transaction or a contract creation for signed transactions.
+
+
+Gateway does not store keys, so transactions sent via Gateway must be signed ahead of time using another provider like [ethers](https://docs.ethers.io/v5/api/signer/) (via `eth_signTransaction`) and sent with `eth_sendRawTransaction`.
+
+## Parameters
+
+`DATA` The signed transaction data.
+
+## Returns
+
+`DATA`, 32 Bytes - the transaction hash, or the zero hash if the transaction is not yet available.
+
+## **Request example**
+
+```bash
+curl --location 'https://rpc.eth.gateway.fm' \
+--header 'Content-Type: application/json' \
+--data '{
+    "id": 6008149,
+    "jsonrpc": "2.0",
+    "method": "eth_sendRawTransaction",
+    "params": [
+        "0xf8687884f46109038252089447ac4f3a5ea94b648672648e730bfe48ed6e734985e8d4a51000802ca0ee5f7c40c98ce6f0f2aa05ebb65e96ddf402f1152fbb8b76893ddb2b0fd6f1d0a068e60320ad7ab56a5200d55ba97c783d4ad37945eb74d1f5d3100155f059fbae"
```

Comment:
This TX returns {"code":-32000,"message":"INTERNAL_ERROR: rlp parse transaction: invalid chainID, 4 (expected 1)"}} 
Means it created from different network. Not ideal example.

Reply:
okay i fixed it